keyboards/fjlabs/fe87/keyboard.json

```
@@ -0,0 +1,133 @@
+{
+    "keyboard_name": "Fe87",
+    "manufacturer": "FJLaboratories",
+    "url": "http://www.fjlaboratories.com/",
+    "maintainer": "mrnoisytiger",
+    "usb": {
+        "vid": "0x7074",
+        "pid": "0x2642",
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
+        "cols": ["F7", "F6", "F5", "F4", "F3", "F2", "F1", "E2", "C7", "C6", "C5", "C4", "C3", "C2", "C1", "D6", "D7"],
+        "rows": ["B3", "B4", "B5", "C0", "E1", "E0"]
+    },
+    "diode_direction": "COL2ROW",
+    "processor": "at90usb646",
+    "bootloader": "atmel-dfu",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "command": true,
+        "nkro": true,
+        "rgblight": false
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "layouts": {
+        "LAYOUT_tkl_all": {
```

Looks like this matches default `tkl_ansi` layout?
```suggestion
    },
    "community_layouts": ["tkl_ansi"],
    "layouts": {
        "LAYOUT_tkl_ansi": {
```

---

@waffle87 Is there somewhere I can get a reference to see what the community layouts look like? 

I'm looking at the QMK documentation, and I'm not seeing any information regarding what the "tkl_ansi" layout actually looks like.

---

> @mrnoisytiger
> 
> > I'm looking at the QMK documentation, and I'm not seeing any information regarding what the "tkl_ansi" layout actually looks like.
> 
> Recommend starting with `/layouts/default/readme.md`

Very helpful! Thank you! Confirmed that this matches `tkl_ansi` based on that readme.